examples/transmission_main_example.py

```diff
@@ -47,40 +47,55 @@ def main(args):
             32e9, 0.15, Power(1e-3, 0, 0)) for f in range(1,97)))
 
     trx = [n for n in network.nodes() if isinstance(n, Transceiver)]
-    source, sink = trx[0], trx[1]
- 
-    path = dijkstra_path(network, source, sink)
-    print(f'There are {len(path)} network elements between {source} and {sink}')
-
-    for el in path:
-        si = el(si)
-        print(el)
-
-    nodelist = [n for n in network.nodes() if isinstance(n, (Transceiver, Fiber))]
-    pathnodes = [n for n in path if isinstance(n, (Transceiver, Fiber))]
-    edgelist = [(u, v) for u, v in zip(pathnodes, pathnodes[1:])]
-    node_color = ['#ff0000' if n is source or n is sink else
-                  '#900000' if n in path else '#ffdfdf'
-                  for n in nodelist]
-    edge_color = ['#ff9090' if u in path and v in path else '#ababab'
-                  for u, v in edgelist]
-    labels = {n: n.location.city if isinstance(n, Transceiver) else ''
-              for n in pathnodes}
-
-    fig = figure()
-    pos = {n: (n.lng, n.lat) for n in nodelist}
-    kwargs = {'figure': fig, 'pos': pos}
-    plot = draw_networkx_nodes(network, nodelist=nodelist, node_color=node_color, **kwargs)
-    draw_networkx_edges(network, edgelist=edgelist, edge_color=edge_color, **kwargs)
-    draw_networkx_labels(network, labels=labels, font_size=14, **kwargs)
-    title(f'Propagating from {source.loc.city} to {sink.loc.city}')
-    axis('off')
-    show()
+    if args.list>=1:
```

Comment:
I don't think listing all the network nodes should be exclusive with running a simulation.

If that's the case, there should be a greater separation between these two modalties.